cosmic-usage-api/src/test/resources/test-public-ips-repository-data.sql

```diff
@@ -0,0 +1,24 @@
+INSERT INTO `domain`
```

Comment:
I'd suggest renaming this file to `public-ips-repository-test-data.sql` and doing the same with other similar files like `test-domains-repository-data.sql`, `test-virtual-machines-repository-data.sql`, etc.